src/App.tsx

```
@@ -1,26 +1,41 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { useEffect, useState } from 'react';
+import { getTodos } from './api';
+import { todosSlice } from './features/todos';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+export const App = () => {
+  const currentTodo = useAppSelector(state => state.currentTodo);
+  const dispatch = useAppDispatch();
 
-          <div className="block">
-            <TodoFilter />
-          </div>
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    getTodos()
+      .then(todos => dispatch(todosSlice.actions.setTodos(todos)))
+      .catch(() => {})
```

Currently the fetch error is swallowed (.catch(() => {})). This makes debugging and automated checks harder. At minimum log the error with `console.error(err)` here, or better, introduce an `error` state (e.g., `const [error, setError] = useState<string | null>(null)`) and set it in the catch to surface a user-visible, non-blocking message. This aligns with the earlier suggestion to avoid `prompt(error)` and not silently ignore failures.